controlpanel/api/models/app.py

```diff
@@ -72,10 +79,22 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
         if is_create:
-            cluster.App(self).create_iam_role()
+            cluster.App(self).create()
 
         return self
 
     def delete(self, *args, **kwargs):
-        cluster.App(self).delete_iam_role()
+        cluster.App(self).delete()
         super().delete(*args, **kwargs)
+
+
+class AddCustomerError(Exception):
+    pass
+
+
+class DeleteCustomerError(Exception):
+    pass
+
+
+App.AddCustomerError = AddCustomerError
```

Comment:
why? just to make it more djangothonic?

Reply:
saves importing a list of names 🤷‍♂ i'm not wed to it, if you think it should go

Reply:
it's fine, no need to change